packages/tools/client-debugger/client-debugger-view/src/components/ContainerSummaryView.tsx

```diff
@@ -132,40 +168,73 @@ export function ContainerSummaryView(props: ContainerSummaryViewProps): React.Re
 			statusComponents.push(connectionStateToString(containerState.connectionState));
 		}
 	}
-	const statusString = statusComponents.join(" | ");
+
+	function ContainerIDRow(label: string, id: string | undefined): React.ReactElement {
+		return (
+			<TableRow>
+				<TableCell {...columnSizing_unstable.getTableCellProps("containerProperty")}>
+					<b>{label}</b>
+				</TableCell>
+				<TableCell>{id}</TableCell>
+			</TableRow>
+		);
+	}
+
+	function ContainerStatusRow(): React.ReactElement {
+		return (
+			<TableRow>
+				<TableCell>
+					<b>Status</b>
+				</TableCell>
+				<TableCell>
+					<TableCellLayout
+						media={((): JSX.Element => {
+							switch (statusComponents[0]) {
+								case "attaching":
+									return (
+										<Badge shape="rounded" color="warning">
+											{statusComponents[0]}
+										</Badge>
+									);
+								case "detached":
+									return (
+										<Badge shape="rounded" color="danger">
+											{statusComponents[0]}
+										</Badge>
+									);
+								default:
+									return (
+										<Badge shape="rounded" color="success">
```

Comment:
The colors here will respect the theme? No need to use `tokens`?

Reply:
Yes, light/dark theme doesn't affect the badge colours